openedx_lti_tool_plugin/deep_linking/views.py

```diff
@@ -69,13 +70,15 @@ def post(
 
         """
         try:
+            # Get launch message.
             message = DjangoMessageLaunch(
                 request,
                 self.tool_config,
                 launch_data_storage=self.tool_storage,
             )
+            # Check launch message type.
             validate_deep_linking_message(message)
-
+            # Redirect to DeepLinkingForm view.
```

Comment:
@kuipumu Add blank line.

Reply:
@anfbermudezme I don't think is necessary to add a blank line